plaso/parsers/systemd_journal.py

```diff
@@ -66,8 +68,8 @@ class SystemdJournalParser(
   _HEADER_SIZE_VERSION_187 = 224
   _HEADER_SIZE_VERSION_189 = 240
   _HEADER_SIZE_VERSION_246 = 256
-  _HEADER_SIZE_VERSION_252 = 268
-  _HEADER_SIZE_VERSION_254 = 276
+  _HEADER_SIZE_VERSION_252 = 264
```

Comment:
fwiw I also have a fix for this here - https://github.com/log2timeline/plaso/pull/4738

Reply:
@sydp given #4738 is a very small PR I would recommend closing it in favor of this one